lib/Sabberworm/CSS/Value/Size.php

```diff
@@ -28,9 +28,11 @@ public static function parse(ParserState $oParserState, $bIsColorComponent = fal
 		if ($oParserState->comes('-')) {
 			$sSize .= $oParserState->consume('-');
 		}
-		while (is_numeric($oParserState->peek()) || $oParserState->comes('.')) {
+		while (is_numeric($oParserState->peek()) || $oParserState->comes('.') || $oParserState->comes('e+', true) || $oParserState->comes('e-', true)) {
```

Comment:
If I’m not mistaken, the `+` isn’t mandatory.

Reply:
True. Fixed.